cmd/cmd.go

```diff
@@ -84,6 +85,9 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 
 	if spinner != nil {
 		spinner.Stop()
+		if spinner.description != "success" {
+			return errors.New("unexpected end to create model")
```

Comment:
does the error from the server still get printed before this returns?

Reply:
It should. The server error is handled and printed in `fn`